mux_test.go

```diff
@@ -2828,6 +2828,40 @@ func TestContextMiddleware(t *testing.T) {
 	r.ServeHTTP(rec, req)
 }
 
+func TestGetVarNames(t *testing.T) {
+	r := NewRouter()
+
+	route := r.Host("{domain}").
+		Path("/{group}/{item_id}").
+		Queries("some_data1", "{some_data1}").
+		Queries("some_data2_and_3", "{some_data2}.{some_data3}")
+
+	// Order of vars in the slice is not guaranteed, so just check for existence
+	expected := map[string]bool{
```

Comment:
Change expected to this:

```go
expected := []string{"domain", "group", "item_id", "some_data1", "some_data2", "some_data3"}
```

Reply:
This is deliberate, since I don't want to enforce any guarantees about slice ordering in the test.